curated_transformers/tests/test_pipe.py

```diff
@@ -85,9 +88,19 @@ def _hf_tokenize_per_token(tokenizer, docs):
     hf_encoding = [tokenizer([token.text for token in doc]) for doc in docs]
     ids = []
     lens = []
+    bos_id = (
+        tokenizer.bos_token_id
+        if tokenizer.bos_token_id is not None
+        else tokenizer.cls_token_id
+    )
+    eos_id = (
+        tokenizer.eos_token_id
+        if tokenizer.eos_token_id is not None
+        else tokenizer.sep_token_id
```

Comment:
What's the reasoning behind falling back to the CLS and SEP token IDs?

Reply:
BERT doesn't define `bos_token_id` and `eos_token_id` and the `[CLS]` and `[SEP]` tokens fulfill a similar role.